File: Lab/W3/ex1/myApp/Coffee.cs
class Coffee : CoffeinBeverage
{
    protected override void brew()
    {
        Console.WriteLine("Dripping Coffee through filter");
    }

    protected override void addCondiments()
    {
        Console.WriteLine("Adding Sugar and Milk");
    }

    protected override bool customerWantsCondiments()
    {
        Console.WriteLine("Would you like milk and sugar with your coffee? (y/n)");
        string answer = Console.ReadLine().ToLower();
        return answer == "yes";
    }
}

File: W3/src/Coffee.java
import java.util.Scanner;

public class Coffee extends CoffinBeverage {
    public void brew(){
        System.out.println("Brewing coffee grinds");
    }
    public void addCondiments() {
        System.out.println("Adding sugar and milk");
    }
    public boolean customerWantsCondiments() {
        String answer = getUserInput();
        if (answer.toLowerCase().startsWith("y")) {
            return true;
        }
        else {return false;}
    }
    public String getUserInput() {
        System.out.print("Do you want to add Condiments? (y/n): ");
        Scanner scanner = new Scanner(System.in);
        return scanner.nextLine();
    }
}
